text/0111-pure-proxy-replication.md

```diff
@@ -0,0 +1,134 @@
+    // proxy_type: ProxyType::Any,
+  } 
+  // ...
+  
+  /// The replication call to be initiated on the source chain.
+  // Simplified version, the XCM part will be abstracted by the `Config` trait.
+  fn replicate(origin: SignedOrigin, witness: Witness, proxy: xcm::Location) -> ... {
+       let pure = ensure_signed(origin);
+       ensure!(pure == proxy_pallet::derive_pure_account(witness), Error::NotPureAccount);
+       let xcm = vec![
+         DescendOrigin(who),
+         Transact(
+             // …
+             origin_kind: OriginKind::Xcm,
+	     call: pallet_proxy_replica::create(witness, proxy).encode(),
+         )
+       ];
+       xcmTransport::send(xcm)?;
+  }
+  // …
+  
+  /// The call initiated by the source chain on the receiving chain.
+  // `Config::CreateOrigin` - generally open for whitelisted parachain IDs and 
+  // converts `Origin::Xcm(ParachainA/AccountId32(pure1))` to `AccountID(pure1)`.
+  fn create(origin: Config::CreateOrigin, witness: Witness, proxy: xcm::Location) -> ... {
+       let pure = T::CreateOrigin::ensure_origin(origin);
+       ensure!(pure == proxy_pallet::derive_pure_account(witness), Error::NotPureAccount);
+       proxy_pallet::create_pure_proxy(pure, proxy);
+  }
+}
+
+```
+
+## Drawbacks
+
+There are two disadvantages to this approach:
+- The receiving chain has to trust the sending chain's claim that the account controlling the pure account has commanded the replication.
+- Clients must obtain witness data.
+
+We could eliminate the first disadvantage by allowing only the spawner of the pure proxy to recreate the pure proxies, if they sign the transaction on a remote chain and supply the witness/preimage. Since the preimage of a pure account includes the account ID of the spawner, we can verify that the account signing the transaction is indeed the spawner of the given pure account. However, this approach would grant exclusive rights to the spawner over the pure account, which is not a property of pure proxies at present. This is why it's not an option for us.
+
+As an alternative to requiring clients to provide a witness data, we could label pure accounts on the source chain and trust it on the receiving chain. However, this would require the receiving chain to place greater trust in the source chain. If the source chain is compromised, any type of account on the trusting chain could also be compromised.
+
+A conceptually different solution would be to not implement replication of pure proxies and instead inform users that ownership of a pure proxy on one chain does not imply ownership of the same account on another chain. This solution seems complex, as it would require UIs and clients to adapt to this understanding. Moreover, mistakes would likely remain unavoidable.
+
+## Testing, Security, and Privacy
+
+Each chain expressly authorize another chain to replicate its pure proxies, accepting the inherent risk of that chain potentially being compromised.
```

Comment:
Would be useful to explain in the RFC the possible scenario if the other chain is compromised